artman/tasks/protoc_tasks.py

```diff
@@ -519,3 +519,24 @@ def execute(self, gapic_api_yaml):
                 continue
             if 'package_name' in go_settings:
                 return go_settings.get('package_name')
+
+
+class JavaProtoCopyTask(task_base.TaskBase):
+    """Copies the .proto files into the grpc_code_dir directory
+    """
+    def execute(self, src_proto_path, grpc_code_dir):
+        grpc_proto_dir = os.path.join(grpc_code_dir, 'src', 'main', 'proto')
+        for proto_path in src_proto_path:
+            index = proto_path.rfind('google')
+            if index == -1:
+                raise ValueError('src_proto_path did not contain "google" '
+                                 'in path as expected. src_proto_path: '
+                                 '"{}"'.format(proto_path))
+
+            src_proto_files = self.exec_command(
```

Comment:
Parsing stdout from `find` is a little strange -- could use [`os.walk`](https://docs.python.org/2/library/os.html#os.walk) instead?

(Note: I'm not actually sure where we've set the boundary between using shell commands and using the Python standard library, but ISTM we should prefer the latter.)